website/i18n/zh/docusaurus-plugin-content-blog/2022/02/21/nacos.md

```diff
@@ -0,0 +1,195 @@
+  curl -X GET 'http://127.0.0.1:8848/nacos/v1/ns/instance/list?serviceName=APISIX-NACOS'
+  ```
+
+正确返回结果示例如下：
+
+  ```JSON
+  {
+    "name": "DEFAULT_GROUP@@APISIX-NACOS",
+    "groupName": "DEFAULT_GROUP",
+    "clusters": "",
+    "cacheMillis": 10000,
+    "hosts": [
+      {
+        "instanceId": "127.0.0.1#3005#DEFAULT#DEFAULT_GROUP@@APISIX-NACOS",
+        "ip": "127.0.0.1",
+        "port": 3005,
+        "weight": 1.0,
+        "healthy": true,
+        "enabled": true,
+        "ephemeral": true,
+        "clusterName": "DEFAULT",
+        "serviceName": "DEFAULT_GROUP@@APISIX-NACOS",
+        "metadata": {},
+        "instanceHeartBeatInterval": 5000,
+        "instanceHeartBeatTimeOut": 15000,
+        "ipDeleteTimeout": 30000,
+        "instanceIdGenerator": "simple"
+      }
+    ],
+    "lastRefTime": 1643191399694,
+    "checksum": "",
+    "allIPs": false,
+    "reachProtectionThreshold": false,
+    "valid": true
+  }
+  ```
+
+### 步骤二：新增 Nacos 路由
+
+使用 Apache APISIX 提供的 Admin API 创建一个新的[路由（Route）](https://apisix.apache.org/zh/docs/apisix/admin-api#route)，APISIX 通过 `upstream.discovery_type` 字段选择使用的服务发现类型，`upstream.service_name` 需要与注册中心的对应服务名进行关联，因此创建路由时指定服务发现类型为 `nacos`。
+
+  ```Shell
+  curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -i -d '
+  {
+      "uri": "/nacos/*",
+      "upstream": {
+          "service_name": "APISIX-NACOS",
+          "type": "roundrobin",
+          "discovery_type": "nacos"
+      }
+  }'
+  ```
+
+在上述命令中，请求头 `X-API-KEY` 是 Admin API 的访问 token，可以在 `conf/config.yaml` 文件中的 `apisix.admin_key.key` 查看。
+
+添加成功后，正确返回结果示例如下：
+
+  ```JSON
+  {
+    "action": "set",
+    "node": {
+      "key": "\/apisix\/routes\/1",
+      "value": {
+        "update_time": 1643191044,
+        "create_time": 1643176603,
+        "priority": 0,
+        "uri": "\/nacos\/*",
+        "upstream": {
+          "hash_on": "vars",
+          "discovery_type": "nacos",
+          "scheme": "http",
+          "pass_host": "pass",
+          "type": "roundrobin",
+          "service_name": "APISIX-NACOS"
+        },
+        "id": "1",
+        "status": 1
+      }
+    }
+  }
+  ```
+
+除此之外，您还可以在 upstream.discovery_args 中传递其他服务相关参数用于指定服务所在的命名空间或组别，具体内容可参考[官方文档](https://apisix.apache.org/zh/docs/apisix/next/discovery/nacos/#%E5%8F%82%E6%95%B0)。
+
+### 步骤三：验证配置结果
+
+使用以下命令发送请求至需要配置的路由。
+
+  ```Shell
+  curl -i http://127.0.0.1:9080/nacos/
+  ```
+
+正常返回结果示例如下：
+
+  ```Apache
+  HTTP/1.1 200 OK
+  Content-Type: text/plain; charset=utf-8
+  Content-Length: 11
+  Connection: keep-alive
+  Date: Thu, 27 Jan 2022 00:48:26 GMT
+  Server: APISIX/2.12.0
+
+  Hello World
+  ```
+
+通过示例看到，Apache APISIX 中新增的路由已经可以通过 Nacos 服务发现找到正确的服务地址，并正常响应。
+
+## 总结
+
+本文为大家介绍了注册中心的概念以及 Apache APISIX 如何配合 Nacos 实现基于服务发现的路由代理。用户可以根据自身的业务需求和过往技术架构使用 Apache APISIX 与 Nacos，以实现接口服务的代理和路由转发的能力。
+关于 `nacos` 插件的更多说明和完整配置信息，可参考[Apache APISIX 官方文档](https://apisix.apache.org/zh/docs/apisix/discovery/nacos)。
```

Comment:
Add a new line to separate them as two paragraphs. But this does not look right in terms of markdown formatting. Refer to the English doc above.